# Disables building FluidSynth's tests and documentation because they cause problems on some platforms.

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -864,8 +864,8 @@ ENABLE_TESTING()
 
 # Process subdirectories
 add_subdirectory ( src )
-add_subdirectory ( test )
-add_subdirectory ( doc )
+
+
 
 # pkg-config support
 set ( prefix "${CMAKE_INSTALL_PREFIX}" )
